docs/index.md

```diff
@@ -35,6 +35,8 @@ Popular tools include:
 * [luacheck](https://github.com/mpeterv/luacheck), another static analyzer, popular outside the Roblox community
 * [ripgrep](https://github.com/BurntSushi/ripgrep), an extremely fast code search tool
 * [Tokei](https://github.com/XAMPPRocky/tokei), a tool for statistics like lines of code
+* [Roact](https://github.com/Roblox/roact), A declarative UI library for Roblox Lua inspired by React
+* [Rodux](https://github.com/Roblox/rodux), A state management library for Roblox Lua inspired by Redux
```

Comment:
I'm hesitant to list these projects because they're not specifically enabled by Rojo. lots of folks use Roact and Rodux but don't use Rojo. The other tools in this list aren't available without Rojo!

Reply:
This is a fantastic point!